src/analyses/tutorials/signs.ml

```diff
@@ -60,7 +60,7 @@ struct
 
   (* This should now evaluate expressions. *)
   let eval (d: D.t) (exp: exp): SL.t = match exp with
-    | Const (CInt64 (i, _, _)) -> SL.top () (* TODO: Fix me! *)
+    | Const (CInt (i, _, _)) -> SL.top () (* TODO: Fix me! *)
```

Comment:
The solution to this tutorial is supposed to use `SL.of_int: int64 -> SL.t` here, but that won't work now with `cilint`. The above `Signs.of_int` should be adapted to do comparisons using `cilint` instead such that this is still as straightforward as before.